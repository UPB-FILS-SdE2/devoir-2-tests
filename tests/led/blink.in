led blink 500 5 3 2
exit