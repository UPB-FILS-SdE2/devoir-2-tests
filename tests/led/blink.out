cmd: Set pixel 3 2 to 9
Sleep for 500 ms
Set pixel 3 2 to 0
Sleep for 500 ms
Set pixel 3 2 to 9
Sleep for 500 ms
Set pixel 3 2 to 0
Sleep for 500 ms
Set pixel 3 2 to 9
Sleep for 500 ms
Set pixel 3 2 to 0
Sleep for 500 ms
Set pixel 3 2 to 9
Sleep for 500 ms
Set pixel 3 2 to 0
Sleep for 500 ms
Set pixel 3 2 to 9
Sleep for 500 ms
Set pixel 3 2 to 0
Sleep for 500 ms
cmd: 